src/onegov/town6/templates/macros.pt

```diff
@@ -834,6 +834,16 @@
                     <li tal:condition="person.website and 'website' not in exclude" class="person-card-website">
                         <a href="${person.website}">${person.website}</a>
                     </li>
+                    <div class="person-card-context-specific-functions" tal:condition="python: len(functions) != 0 and len(organizations) != 0 and organizations is not None and functions is not None">
+                        <div tal:define="map python: zip(organizations, functions)">
+                            <span i18n:translate>Functions:</span>
+                            <ul>
+                                <li tal:repeat="(organization, function) map">
+                                    <span>${organization}: ${function}</span>
```

Comment:
you check if function is empty above, I guess this means that there will be cases rendered as organization without function, e.g. "Organization:"?

Reply:
Yes, sometimes there exists an organization, but no mapping to function. (The function field is optional)